R/load_understat.R

```diff
@@ -50,3 +51,8 @@ load_understat_league_shots <- function(league) {
   return(dat_df)
 
 }
+
+check_league_name <- function(league_name) {
+  leagues <- names(LEAGUES)
+  if(!league_name %in% leagues) stop("Check league name")
```

Comment:
```suggestion
  arg_match0(league_name, names(LEAGUES))
```

Reply:
If we're going to improve the error handling, then we should really improve it. `arg_match0` specifies what the allowed values are when erroring.

Reply:
The `arg_match0()` function is great. Thank you very much for recommending it 🙌🏿.